### PR TITLE
Clarify PEP517 step

### PR DESCRIPTION
**Pull request by Kangie**
Update lists to use autonumbering

### CODE REVIEW

**Comment by Kangie** [comment]
As I understand it, PEP 517 mode should not be used with  ``distutils_install_for_testing`` or  ``distutils_enable_tests`` with ``--install`` - emerge now throws errors:

```
distutils_enable_tests --install is not implemented in PEP517 mode
distutils_install_for_testing is not implemented in PEP517 mode
```
This should avoid any confusion going forward and make changing list items more maintainable by using autonumbering.

**Comment by mgorny** [comment]
I dare say it's not obsolete, given that there's still a large number of packages that don't use PEP517 mode yet.

**Comment by mgorny** [comment]
Note that there's **or** in there. I suppose the two other options are kinda obsolete now but the sentence is still correct.

**Comment by Kangie** [comment]
It might just be the way that it's phrased - when I read:

```
3. Switch to PEP 517 mode, add ``distutils_install_for_testing``
   to the test sub-phase or ``--install`` to ``distutils_enable_tests``
   call.  This resolves majority of problems with the test suite
   requiring the package to be installed prior to testing.
```
I interpret that as:

1. Enable `DISTUTILS_USE_PEP517`, then

do either of:

- add `d_i_f_t`  to `python_test` (which errors)
- add `--install` to `distutils_enable_tests` (which errors).

Is there a better way to phrase this, because at the very least *I* seem to be missing something? :)

Happy to have a go at updating the wording if you can point me in the right direction.

Edit: I think I see it, it's

do any of:

- Enable `DISTUTILS_USE_PEP517`
- add `d_i_f_t`  to `python_test` || add `--install` to `distutils_enable_tests`

File: test.rst [hunk 1]
```diff
@@ -127,53 +127,53 @@ Checklist for dealing with test failures
...
+#. Try running the test suite from another directory.  If you're seeing
    failures to load compiled extensions, Python may be wrongly importing
    modules from the current directory instead of the build/install tree.
    Some test suite also depend on paths relative to where upstream run
    tests.
 
-3. Switch to PEP 517 mode, add ``distutils_install_for_testing``
-   to the test sub-phase or ``--install`` to ``distutils_enable_tests``
-   call.  This resolves majority of problems with the test suite
+#. Either switch to PEP 517 mode, or add ``distutils_install_for_testing``
```
Comment (mgorny) [inline test.rst]: Maybe add '(preferred)' after PEP 517 mode.

**Comment by mgorny** [comment]
Also please revert to normal numbering.

**Review by mgorny — APPROVED** [review]
Thanks a lot!

**Pull request closed without merging**